packages/backend/src/open_payments/invoice/service.ts

```diff
@@ -200,34 +185,34 @@ async function handleDeactivated(
   deps: ServiceDependencies,
   invoice: Invoice
 ): Promise<void> {
-  assert.ok(invoice.webhookId)
-  const amountReceived = await deps.accountingService.getTotalReceived(
-    invoice.id
-  )
-  if (!amountReceived) {
-    deps.logger.warn(
-      { amountReceived },
-      'invoice with webhook id and empty balance'
+  assert.ok(invoice.processAt)
+  try {
+    const amountReceived = await deps.accountingService.getTotalReceived(
+      invoice.id
     )
-    await invoice.$query(deps.knex).patch({ webhookId: null })
-    return
-  }
+    if (!amountReceived) {
+      deps.logger.warn(
+        { amountReceived },
+        'invoice with processAt and empty balance'
+      )
+      await invoice.$query(deps.knex).patch({ processAt: null })
+      return
+    }
 
-  deps.logger.trace(
-    { amountReceived },
-    'withdrawing deactivated invoice balance'
-  )
-  const error = await deps.accountingService.createWithdrawal({
-    id: invoice.webhookId,
-    accountId: invoice.id,
-    amount: amountReceived,
-    timeout: BigInt(deps.webhookService.timeout) * BigInt(1e9) // ms -> ns
-  })
-  if (error) throw error
+    deps.logger.trace(
+      { amountReceived },
+      'withdrawing deactivated invoice balance'
+    )
+    const error = await deps.accountingService.createWithdrawal({
+      id: invoice.id,
+      accountId: invoice.id,
+      amount: amountReceived,
+      timeout: BigInt(deps.webhookService.timeout) * BigInt(1e9) // ms -> ns
```

Comment:
If `deps.webhookService.timeout` is milliseconds, doesn't `* 1e9` convert to picoseconds?